packages/common-integration-tests/src/common-int-test-utils.ts

```diff
@@ -10,15 +11,24 @@ import {
 } from '@gomomento/sdk-core/dist/src/messages/responses/response-base';
 import {v4} from 'uuid';
 
+export function isLocalhostDevelopmentMode(): boolean {
+  const useLocalhost = process.env.MOMENTO_SDK_TESTS_USE_LOCALHOST;
+  return useLocalhost !== undefined;
+}
 export function testCacheName(): string {
-  const name = process.env.TEST_CACHE_NAME || 'js-integration-test-default';
-  return name + v4();
+  return process.env.TEST_CACHE_NAME || `js-integration-test-default-${v4()}`;
```

Comment:
This change is responsible for the integration test failure

Reply:
thx